src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\LanguageConstruct;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ */
+final class SingleSpaceAfterConstructFixer extends AbstractFixer
+{
+    /**
+     * @var int[]
+     */
+    private static $tokenKinds = [
+        T_BREAK,
+        T_CONTINUE,
+        T_ECHO,
+        T_NEW,
+        T_PRINT,
+        T_RETURN,
+        T_THROW,
```

Comment:
Could list a bunch of additional statements here, however, some of these statements are already handled by other fixers, for example:

```diff
diff --git a/src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php b/src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php
index d8864775..2a70edb7 100644
--- a/src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php
+++ b/src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php
@@ -30,8 +30,15 @@ final class SingleSpaceAfterConstructFixer extends AbstractFixer
         T_BREAK,
         T_CONTINUE,
         T_ECHO,
+        T_ELSE,
+        T_ELSEIF,
+        T_IF,
+        T_INCLUDE,
+        T_INCLUDE_ONCE,
         T_NEW,
         T_PRINT,
+        T_REQUIRE,
+        T_REQUIRE_ONCE,
         T_RETURN,
         T_THROW,
     ];
```

Reply:
I think that for consistency, all of these should be added to the fixer. This should probably even belong to the `@PSR2` ruleset, and then all of the other fixers would not need to worry about whitespace adjustment.

What do you think?

Reply:
Indeed, PSR-2 requires to have a single space after most of those keywords:
> Control structure keywords MUST have one space after them

Though I don't think this applies to e.g. `echo` or `print`. Maybe the list should be configurable so that the `@PSR2` ruleset enables the fixer with the tokens related to this rule only?

Reply:
Sounds good, @julienfalque!